### PR TITLE
added footer to the project

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
- Added footer to the project.

## Screenshots
image

## Trello Card
https://trello.com/c/Oozy2mKS

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -128,6 +128,13 @@ <h3 class="m-bottom-1">Django</h3>
           </article>
         </div>
       </section>
+      <footer class="footer-project">
```
Comment (alejobh) [inline maquetado/index.html]: same as the previous PR, try to let `<nav>` and `<footer>` in the same level of the `<body>`

File: maquetado/scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0;
+  position: relative;
+  width: 100%;
+}
+
+.footer-title,
+.footer-copyright {
```
Comment (alejobh) [inline maquetado/scss/footer.scss]: try to create `footer-text` class or something like that and use it in the navbar texts

File: maquetado/scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+.footer-title {
+  font-size: $text-big;
+}
+
+.footer-copyright {
+  bottom: 0;
+  font-size: $text-small;
+  position: absolute;
+  right: 10px;
```
Comment (alejobh) [inline maquetado/scss/footer.scss]: i think you could stop using `position: absolute and relative` and use flex for this, what do you think if you use `align-self: end` 👍

Comment (DavidRojas1612) [reply maquetado/scss/footer.scss]: I had tried but without result 🙄, now it works. 😅

File: maquetado/scss/footer.scss [hunk 4]
```diff
@@ -18,8 +17,5 @@
 }
 
 .footer-copyright {
-  bottom: 0;
-  font-size: $text-small;
```
Comment (alejobh) [inline maquetado/scss/footer.scss]: are you sure removing the font size its ok?

File: maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.footer-project {
```
Comment (alejobh) [inline maquetado/scss/footer.scss]: i recommend to change this name for `footer` alone

**Pull request merged**